Unit tests fail due to surefire plugin issue with Debian/Ubuntu Java8

See https://issues.apache.org/jira/browse/SUREFIRE-1588

The workaround is to update `maven-surefire-plugin` to version 3.0.0-M1.

```bash
$ mvn clean install
...
[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ piccolo2d-core ---

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Error: Could not find or load main class org.apache.maven.surefire.booter.ForkedBooter

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Piccolo2D .......................................... SUCCESS [  0.270 s]
[INFO] Piccolo2D Core ..................................... FAILURE [  2.019 s]
[INFO] Piccolo2D Extras ................................... SKIPPED
[INFO] Piccolo2D SWT ...................................... SKIPPED
[INFO] Piccolo2D Examples ................................. SKIPPED
[INFO] Piccolo2D SWT Examples ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.850 s
[INFO] Finished at: 2018-11-27T16:56:21-06:00
[INFO] Final Memory: 22M/391M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project piccolo2d-core: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /working/piccolo2d.java/core && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -jar /working/piccolo2d.java/core/target/surefire/surefirebooter5808598591354707365.jar /working/piccolo2d.java/core/target/surefire/surefire174509725156692170tmp /working/piccolo2d.java/core/target/surefire/surefire_07292428692929199916tmp
[ERROR] -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -260,7 +260,7 @@
         </plugin>
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.19.1</version>
+          <version>3.0.0-M1</version>
         </plugin>
         <plugin>
           <groupId>org.apache.felix</groupId>

